content/blog/2023-12-11-circ-bitwidth-adjustment

```diff
@@ -0,0 +1,115 @@
++++
+title = "CirC Bitwidth Adjustment Implementation"
+[extra]
+bio = """
+  Collin Zhang and Melinda Fang are students at Cornell
+"""
+[[extra.authors]]
+name = "Collin Zhang"
+link = "https://collinzrj.github.io/"  # Links are optional.
+[[extra.authors]]
+name = "Melinda Fang"
++++
+
+## Link to the project
+We are currently working on creating a PR to include the change to the project. At this point, the best way to see the changes we have made is via this link which compares the latest commit with the commit before our changes https://github.com/collinzrj/circ/compare/a26533baadfd674f7d7d766b995a9a78a02d4ebd..61fb7c0ee2946cf84b92d962236ffe9533cb6c53
```

Comment:
Instead of listing this right at the top of the post, how about including it more elegantly as a link somewhere in the body text? Like, maybe when you introduce the idea for the first time, you can mention what the current status is and link to your code there.